### PR TITLE
Disable omitempty for storage contract types

### PR DESCRIPTION
**Pull request by taiyangc**
Since we display storage contract types on command line, we do not want to omit the empty values (0) which may be valid.

This involved a protobuf update here: https://github.com/TRON-US/protobuf/pull/1

Locally you need to run `make install` with the latest Makefile.

### CODE REVIEW

File: protos/node/node.proto [hunk 1]
```diff
@@ -128,7 +132,9 @@ message ContractStat {
 }
 
 message Contracts {
+  option (gogoproto.json_no_omit_empty) = true;
```
Comment (JustinGaoF) [inline protos/node/node.proto]: suggest just use guard's contractMeta here,  as that contract should contains all information for the contract, and in the future, renter/host can use the signature of that contract to prove the contract's existence.

Comment (taiyangc) [reply protos/node/node.proto]: This pr has nothing to do with the creation of contracts, it's simply a modification of marshal types. Can you mention the original design of not using contractMeta directly? @wxue

Comment (wxue) [reply protos/node/node.proto]: @JustinGaoF That's a good suggestion, but the current contractMeta in Guard is not "meta" enough -- meaning, the contract here refers per file, not per shard, and this is mainly for UI/UX feed: https://docs.google.com/document/d/1OYMX4T-s_UexLooyo4Ldhj0rpN8Mh9DvurenLFTy4Zg/edit#heading=h.7w6f8xmxl4vj. 

Do you think we should make a reusable contractMeta with the most common basic info in it? - So node proto can also use it.
Will that create a big workload for the guard(/escrow)?

Comment (taiyangc) [reply protos/node/node.proto]: I suggest doing a change later, perhaps when renter UI/UX design is finalized, that should help us see how "meta" the contracts message should get.

Comment (wxue) [reply protos/node/node.proto]: sounds good

Comment (JustinGaoF) [reply protos/node/node.proto]: +1

**Pull request merged**